backend/src/main/java/com/festago/auth/application/AuthService.java

```diff
@@ -45,7 +46,7 @@ private UserInfo getUserInfo(LoginRequest request) {
     }
 
     private String getAccessToken(Member member) {
-        return authProvider.provide(new AuthPayload(member.getId()));
+        return authProvider.provide(new AuthPayload(member.getId(), Role.MEMBER));
```

Comment:
Member에 Role 컬럼이 생긴다면 다음과 같이 될 것 같습니다.
```suggestion
        return authProvider.provide(new AuthPayload(member.getId(), member.getRole()));
```